src/frontend/admin/src/components/presentational/table/TableComponent.tsx

```diff
@@ -199,9 +199,6 @@ export function TableComponent<T extends GridValidRowModel>({
         <DataGrid
           {...props}
           paginationMode={paginationMode}
-          getRowHeight={() => {
-            return 40;
-          }}
```

Comment:
It increases row height everywhere. Particularly in list table and now I feel the row height is too high in those lists. Are you able to set row height only for table in list view ?

| Before | After |
|--------|-------|
|<img width="800" alt="image" src="https://github.com/openfun/joanie/assets/9265241/8738d28a-b0c6-495d-90f4-405982f77f28">|<img width="800" alt="image" src="https://github.com/openfun/joanie/assets/9265241/ec18dd84-4d16-4ea6-a8c0-162fa679ca23">|